src/web/ng/Connection.hpp

```diff
@@ -105,7 +105,7 @@ class Connection : public ConnectionMetadata {
     /**
      * @brief The default timeout for send, receive, and close operations.
      */
-    static constexpr std::chrono::steady_clock::duration DEFAULT_TIMEOUT = std::chrono::seconds{30};
+    static constexpr std::chrono::steady_clock::duration DEFAULT_TIMEOUT = std::chrono::seconds{10};
```

Comment:
I don't see this change in PR summary. Is it typo? or intended?

Reply:
This is intended. I thought it is better to disconnect inactive clients earlier.